vividus-bdd-engine/src/main/java/org/vividus/bdd/model/jbehave/Scenario.java

```diff
@@ -64,9 +58,9 @@ public void setExamples(Examples examples)
 
     public List<Step> collectSteps()
     {
-        if (steps != null)
+        if (getSteps() != null)
         {
-            return steps;
+            return getSteps();
         }
         return Optional.ofNullable(examples)
```

Comment:
also I believe you should somehow consider before/after scenario steps of Examples

Reply:
this method is only needed to get steps that go into Xray scenario

Reply:
can rename it to collectScenarioSteps